### PR TITLE
v0.7.0

### PR DESCRIPTION
**Pull request by peterkeating**
- Various updates to "Section" 
- Remove widget template overrides as moved to widget module
- Tweak to paragraph margin
- Various improvements to "Hero" widget
- Apply max width constraint to video within light gallery
- Improvements to "Footer" widget
- Various improvements to header
- Add "Navigation" widget
- Add "ResponsiveImage" widget

### CODE REVIEW

File: content/Assets/Styles/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+/* manuale-700 - latin */
+@font-face {
+    font-family: 'Manuale';
+    font-style: normal;
+    font-weight: 700;
+    src: local(''), url('../Fonts/manuale-v8-latin-700.woff2') format('woff2'),
+        /* Chrome 26+, Opera 23+, Firefox 39+ */
+            url('../Fonts/manuale-v8-latin-700.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
```
Comment (Newnab) [inline content/Assets/Styles/_fonts.scss:19]: All looks good to me, but one thought here: Wouldn't we want to include an italic version for both fonts? Given that CMS users have the option, if we don't provide one the browser will attempt a faux-italic which usually looks gross.

Comment (peterkeating) [reply content/Assets/Styles/_fonts.scss:19]: Good shout.

Comment (Newnab) [reply content/Assets/Styles/_fonts.scss:19]: This conversation was resolved but I can't see any change, is that our intent? If it's deliberate that's cool.

Comment (peterkeating) [reply content/Assets/Styles/_fonts.scss:19]: I've moved it back to unresolved, must have marked it as resolved by mistake.

**Pull request merged**